transcom/mymove
# B 18670 int

## [B-18670](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18670)

## Summary

This task adds the ability to view the information added in tickets [B-18840](https://github.com/transcom/mymove/pull/12260/) and [B-18669](https://github.com/transcom/mymove/pull/12261/) (main PRs linked, includes a link to INT prs on page)

Is there anything you would like reviewers to give additional scrutiny?
Please ensure that the weight is formatted properly; Also that when no weight is given, it shows a dash like other fields around it.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access any HHG shipment as a TOO.
2. Using devtools, find the shipment id in the network tab of the dev console.
3. 5. Scroll down and verify that the Actual pro gear weight and Actual spouse pro gear weight labels and the absence of values is properly formatted (assuming that this shipment didn't have pro gear weight attached to it yet).
4. **Either** update the actualProGearWeight and actualSpouseProGearWeight fields as described [here in B-18669](https://github.com/transcom/mymove/pull/12261/) (main PRs linked, includes a link to INT prs on page) , **OR** if you have access to your database, just look up the MTOShipment by it's id and populate the fields with an integer.
5. While logged in as the TOO viewing the updated shipment, click the "Move Task Order" tab.
6. Scroll down and verify that the Actual pro gear weight and Actual spouse pro gear weight labels and values are present and properly formatted.
7. Ensure frontend tests passing.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
Without change:
<img width="636" alt="image" src="https://github.com/transcom/mymove/assets/148906585/f13be831-e4eb-4338-915b-8ef9c7965a28">
With change:
![image](https://github.com/transcom/mymove/assets/148906585/d5644323-4da0-43a5-8b2d-b7290828eee0)
